std/uni.d

```diff
@@ -3978,7 +3978,7 @@ public:
     }
 }
 
-/*
+/**
```

Comment:
Okay let me rephrase it - it was never _intended_ to be public. The whole thing was too specialized to be of use to wide audience.